spinalcordtoolbox/deepseg/models.py

```diff
@@ -122,12 +122,20 @@
     #       - Binarization is applied within SCT code
     "model_seg_sc_contrast_agnostic_softseg_monai": {
         "url": [
-            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/v2.5/model_contrast-agnostic_20240930-1002.zip"
+            "https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/releases/download/r20241113/model_contrast_agnostic_20241018-0508.zip"
         ],
         "description": "Spinal cord segmentation agnostic to MRI contrasts using MONAI-based nnUNet model",
         "contrasts": ["any"],
         "thr": 0.5,  # Softseg model -> threshold at 0.5
         "default": False,
+        "citation": textwrap.dedent("""
+                                    #######################################################################
+                                    Please cite the following paper when using the contrast-agnostic model:
+
+                                    Bédard*, S., Karthik*, E.N., Tsagkas, C., Pravatà, E., Granziera, C., Smith, A.C., Weber, K.A., & Cohen-Adad, J. (2023).
+                                    Towards contrast-agnostic soft segmentation of the spinal cord. ArXiv, abs/2310.15402. https://arxiv.org/abs/2310.15402
+                                    #######################################################################
+                                """),
```

Comment:
It turns out that we had similar thoughts! I had prototyped a `citation` field in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4685, too!! I think it is a very good idea, and we should strive to add this for all our models. :hearts:

How do you feel about using the bibtex format instead? Similar to what was done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4685/commits/1167734cc07fdb25b06646d7a6b162ce717ea507.

Also, I think perhaps the "Please cite the following paper" text should live inside of `sct_deepseg`, so that we don't duplicate it for every single model. That way, `"citation"` contains only the citation and no other extraneous text.